### PR TITLE
doctor page #72

### PR DESCRIPTION
**Pull request by mohannadhanafi**
relates #72

### CODE REVIEW

File: src/views/doctors.hbs [hunk 1]
```diff
@@ -10,7 +10,7 @@
                     <span class="doctor_age">Age: 45</span>
```
Comment (MohammedYehia) [inline src/views/doctors.hbs]: why is this a static value and not a dynamic like the other??

File: src/controllers/index.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ const articles = require('./articles');
 const doctors = require('./doctors');
 const signUp = require('./signUp');
 const signIn = require('./signIn');
+const doctor = require('./doctor');
```
Comment (MohammedYehia) [inline src/controllers/index.js]: where is this doctor.js controller ??

Comment (MohammedYehia) [reply src/controllers/index.js]: this is in your local machine but I can't find the file in your PR files

Comment (mohannadhanafi) [reply src/controllers/index.js]: It exists now :+1:

**Pull request merged**